tracing/src/span.rs

```diff
@@ -720,10 +720,41 @@ impl Span {
     ///
     /// If this span is disabled, or the resulting follows-from relationship
     /// would be invalid, this function will do nothing.
-    pub fn follows_from(&self, from: impl for<'a> Into<Option<&'a Id>>) -> &Self {
+    ///
+    /// # Example
```

Comment:
there are multiple examples here:
```suggestion
    /// # Examples
```